src/container/srv_target.c

```diff
@@ -772,9 +774,21 @@ cont_child_started(struct ds_cont_child *cont_child)
 	return !d_list_empty(&cont_child->sc_link);
 }
 
+static int cont_close_hdl(uuid_t cont_hdl_uuid);
+
 static void
 cont_child_stop(struct ds_cont_child *cont_child)
```

Comment:
a bit confused here, If the target is excluded from the pool, who will call cont_child_stop?